packages/gatsby-theme-docs/src/components/side-by-side.js

```diff
@@ -3,38 +3,28 @@ import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
 import { css } from '@emotion/core';
 import { designSystem } from '@commercetools-docs/ui-kit';
+import SpacingsStack from '@commercetools-uikit/spacings-stack';
 import { usePageData } from '../hooks/use-page-data';
 
 /**
  * Allows the first and second child of this container to float side by side in large desktop sizes.
- *
- * NOTE about the `!important` override.
- * The <SideBySide> content is sometimes an entry in the UI Kit stack (e.g. in API docs).
- * The UI kit stack forces the top spacing and uses `!important`.
- * However, given that the SideBySide is responsive, the second item can sometimes be
- * to the right of the first and sometimes under it.
- * Therefore, we need different spacing "logic" here - the UI Kit stacks and inlines are
- * not intended for responsive content. As a consequence, without this override, the right
- * item would not be top aligned with the left item.
```

Comment:
Any reason for removing this comment?

Reply:
> Any reason for removing this comment?

Nikolaus wrote it and removed it in his PR. See my comment here - https://github.com/commercetools/commercetools-docs-kit/pull/594/files#r475474449